backend/webui_service/README.md

```diff
@@ -0,0 +1,32 @@
+<!--
+SPDX-License-Identifier: Apache-2.0
+Copyright 2024 Canonical Ltd.
+-->
+
+# Swagger UI Service
+
+The webconsole can optionally serve a [swagger UI](https://github.com/swaggo/swag).
+The API documentation is automatically generated from code annotations.
+
+To generate the swagger UI files run:
+```
+swag init -g backend/webui_service/swagger_ui_service.go --outputTypes go
+```
+The `docs.go` file will automatically be created in `webconsole/docs`
+
+The swagger UI runs by default on `localhost`. If the webconsole server runs remotely, set the following environment variable.
+```
+export SWAGGER_HOST=<webconsole-ip>
+```
+Build the webconsole including the UI option:
+```
+make webconsole-ui
+```
+Access the swagger UI at:
+```
+http://<webconsole-ip>:5000/swagger/index.html
+```
+The `doc.json` file, which can be integrated in a frontend implementation, is available at:
+```
+http://<webconsole-ip>:5000/swagger/doc.json
+```
```

Comment:
Is this available only for developer or SD-Core user as well ? 
Do we ship the doc with regular container image?

Reply:
This is available for developers and for sd-core users. 
At the moment, this functionality works only for the build using the ui tag (make webconsole-ui) but if we wish it is possible to make it the default behavior.
The swagger UI tool for go, automatically creates a web page that is served at `http://<webconsole-ip>:5000/swagger/index.html`. And it automatically creates the `doc.json` file and serves it.